package.json

```diff
@@ -27,7 +27,7 @@
         "express-openapi-validator": "^1.6.2",
         "got": "^9.6.0",
         "helmet": "^3.21.2",
-        "json-translator": "github:CriminalInjuriesCompensationAuthority/json-translator#v0.1.0",
+        "json-translator": "github:CriminalInjuriesCompensationAuthority/json-translator",
```

Comment:
happy to discuss it this is the correct approach?

Reply:
We typically point to a specific version and don't necessarily reference the latest cut of the code.